code/datums/perks/job.dm

```diff
@@ -27,8 +28,9 @@
 	..()
 
 /datum/perk/vagabond
-	name = "Vagabond"
-	desc = "You're used to see the worst sight the world has to offer. Your mind feels more resistant."
+	name = "Drifter"
+	desc = "You're used to see the worst sight the world has to offer. Your mind feels more resistant. \
```

Comment:
Line 32 has a grammatical mistake. "You're used to **seeing** the worst **sights**..."